cmd/agent/subcommands/flare/command.go

```diff
@@ -341,6 +350,22 @@ func makeFlare(flareComp flare.Component,
 		return err
 	}
 
+	enableStreamLog, settningErr := rcsetting.GetRuntimeSetting("enable_stream_logs")
+
+	if settningErr != nil {
+		fmt.Fprintln(color.Output, color.RedString(fmt.Sprintf("Error getting runtime setting: %s", settningErr)))
+	}
+
+	if values, ok := enableStreamLog.([]interface{}); ok && len(values) > 1 {
+		if enable, ok := values[1].(bool); ok && enable {
+			fmt.Fprintln(color.Output, color.GreenString((fmt.Sprintf("Asking the agent to stream logs for %s", defaultRCDuration))))
+			err := streamlogs.StreamLogs(lc, config, &rcStreamLogParams)
+			if err != nil {
+				fmt.Fprintln(color.Output, color.RedString(fmt.Sprintf("Error streaming logs: %s", err)))
+			}
+		}
+	}
+
 	if streamLogParams.Duration > 0 {
 		fmt.Fprintln(color.Output, color.GreenString((fmt.Sprintf("Asking the agent to stream logs for %s", streamLogParams.Duration))))
 		err := streamlogs.StreamLogs(lc, config, &streamLogParams)
```

Comment:
It is unexpected to have two `streamlogs.StreamLogs` calls in this function.
I think it would make sense to have a single one as this two cases seems to be exclusive. Command line can take precedence over `enable_stream_logs` runtime config.